target/linux/ath79/dts/qca9533_mikrotik_routerboard-wapr-2nd.dts

```diff
@@ -50,6 +50,12 @@
 	};
 };
 
+&wmac {
+	status = "okay";
+
+	qca,no-eeprom;
```

Comment:
This is now included in more files. I think you have to activate this in more files. 
````
target/linux/ath79/dts/qca9533_mikrotik_routerboard-lhg-hb.dtsi:#include "qca9533_mikrotik_routerboard-16m.dtsi"
target/linux/ath79/dts/qca9533_mikrotik_routerboard-mapl-2nd.dts:#include "qca9533_mikrotik_routerboard-16m.dtsi"
target/linux/ath79/dts/qca9533_mikrotik_routerboard-wap-2nd.dts:#include "qca9533_mikrotik_routerboard-16m.dtsi"
target/linux/ath79/dts/qca9533_mikrotik_routerboard-wapr-2nd.dts:#include "qca9533_mikrotik_routerboard-16m.dtsi"
```

Reply:
Damn, I didn't think of new devices being added. Thanks for spotting this! I'm fixing it and rebasing.

Reply:
Thanks for your review, @hauke.

I added the removed `&wmac` element to the wireless-capable new devices supported after the rebase.